x/feemarket/types/params.go

```diff
@@ -189,12 +189,12 @@ func validateMinGasMultiplier(i interface{}) error {
 		return fmt.Errorf("invalid parameter: nil")
 	}
 
-	if v.IsZero() || v.IsNegative() {
```

Comment:
Why would one want to multiply the gas by 0?

Reply:
```
max(gasLimit * minGasMultiplier, gasUsed)
```

When `minGasMultiplier` is zero, it's just the old behavior.